core-bundle/contao/classes/DataContainer.php

```diff
@@ -1732,6 +1732,11 @@ public function generateRecordLabel(array $row, string $table = null, bool $prot
 	 */
 	protected static function preloadCurrentRecords(array $ids, string $table): void
 	{
+		if (empty($ids))
```

Comment:
```suggestion
		if (!$ids)
```
`$ids` is always defined here, so there is no need to use the error-suppressing `empty()` I think.

Reply:
I find that much more readable 🤷

Reply:
I think we have agreed on using `!\count($ids)` in this case, haven‘t we?

Reply:
I always forget 😞
One day I will write a CS fixer for that ☺️